### PR TITLE
:readme: add talk show and deck / bdx.io

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,9 @@
 
 # Demo
 [![todomvc](https://image.noelshack.com/fichiers/2017/44/7/1509875726-todo.png)](https://alakarteio.github.io/todomvc/)
+
+## Talk
+ - [BDX.io 2017](https://www.youtube.com/watch?v=SHrSdNzQcXM&index=19&list=PLUJzERpatfsW5PbUcMAIOIqfCoqS7K1F1)
+
+## Slides
```
Comment (fabienjuif) [inline README.md]: Decks

File: README.md [hunk 2]
```diff
@@ -3,3 +3,9 @@
 
 # Demo
 [![todomvc](https://image.noelshack.com/fichiers/2017/44/7/1509875726-todo.png)](https://alakarteio.github.io/todomvc/)
+
+## Talk
```
Comment (fabienjuif) [inline README.md]: Talks

**Review by fabienjuif — CHANGES_REQUESTED** [review]
comments

File: README.md [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
+  <a href="https://alakarteio.github.io/todomvc/" target="_blank">
+    <img src="https://image.noelshack.com/fichiers/2017/44/7/1509875726-todo.png" />
+  </a>
+</p>
 
-## Talk
- - [BDX.io 2017](https://www.youtube.com/watch?v=SHrSdNzQcXM&index=19&list=PLUJzERpatfsW5PbUcMAIOIqfCoqS7K1F1)
+# Talk
+## Who
+ - alakarte [[Twitter](https://twitter.com/alakarteio) | [Github](https://github.com/alakarteio)]
```
Comment (fabienjuif) [inline README.md]: This is not the right place to put this

File: README.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
-# Todo MVC with ReactJS + redux-saga + Redux
+# Todo MVC with ReactJS + Redux-saga + Redux
```
Comment (fabienjuif) [inline README.md]: This is either `redux-saga` or `Redux-Saga`: https://github.com/redux-saga/redux-saga

File: README.md [hunk 5]
```diff
@@ -1,5 +1,24 @@
...
 [![CircleCI](https://circleci.com/gh/alakarteio/todomvc.svg?style=svg)](https://circleci.com/gh/alakarteio/todomvc) [![Coverage Status](https://coveralls.io/repos/github/alakarteio/todomvc/badge.svg?branch=master)](https://coveralls.io/github/alakarteio/todomvc?branch=master)
 
-# Demo
-[![todomvc](https://image.noelshack.com/fichiers/2017/44/7/1509875726-todo.png)](https://alakarteio.github.io/todomvc/)
+# Application
+<p align="center">
+  <a href="https://alakarteio.github.io/todomvc/" target="_blank">
+    <img src="https://image.noelshack.com/fichiers/2017/44/7/1509875726-todo.png" />
```
Comment (fabienjuif) [inline README.md]: this screenshot refer to `Alakarte` not `alakarte`

**Review by fabienjuif — CHANGES_REQUESTED** [review]
comments

File: README.md (hunk 3 above)
Comment (guillaumecrespel) [reply README.md]: :ok_hand:

File: README.md (hunk 4 above)
Comment (guillaumecrespel) [reply README.md]: :ok_hand:

File: README.md (hunk 5 above)
Comment (fabienjuif) [reply README.md]: @guillaumecrespel what about this ?

Comment (guillaumecrespel) [reply README.md]: My OS not show emoji ... i take screenshot later this evening

**Review by fabienjuif — APPROVED** [review]
👌

**Pull request merged**